parlai/tasks/convai2/__init__.py

```diff
@@ -1,5 +1,7 @@
-# Copyright (c) 2017-present, Facebook, Inc.
-# All rights reserved.
-# This source code is licensed under the BSD-style license found in the

```

Comment:
what happened with this one?

Reply:
Especially when the script literally just read the contents of a file and then rewrote them. Perhaps this file was encoded weirdly?